lib/src/model/data.dart

```diff
@@ -34,33 +34,33 @@ Future<void> addRestaurant(Restaurant restaurant) {
 }
 
 Stream<QuerySnapshot> loadAllRestaurants() {
-  return Firestore.instance
+  return FirebaseFirestore.instance
       .collection('restaurants')
       .orderBy('avgRating', descending: true)
       .limit(50)
       .snapshots();
 }
 
 List<Restaurant> getRestaurantsFromQuery(QuerySnapshot snapshot) {
-  return snapshot.documents.map((DocumentSnapshot doc) {
+  return snapshot.docs.map((DocumentSnapshot doc) {
     return Restaurant.fromSnapshot(doc);
   }).toList();
 }
 
 Future<Restaurant> getRestaurant(String restaurantId) {
-  return Firestore.instance
+  return FirebaseFirestore.instance
       .collection('restaurants')
-      .document(restaurantId)
+      .doc(restaurantId)
       .get()
       .then((DocumentSnapshot doc) => Restaurant.fromSnapshot(doc));
 }
 
 Future<void> addReview({String restaurantId, Review review}) {
   final restaurant =
-      Firestore.instance.collection('restaurants').document(restaurantId);
-  final newReview = restaurant.collection('ratings').document();
+  FirebaseFirestore.instance.collection('restaurants').doc(restaurantId);
+  final newReview = restaurant.collection('ratings').doc();
 
-  return Firestore.instance.runTransaction((Transaction transaction) {
+  return FirebaseFirestore.instance.runTransaction((Transaction transaction) {
```

Comment:
This one doesn't work, I think it needs some more to get the transaction going:

```
Error: [cloud_firestore/unknown] Invalid argument (dartObject): Could not convert: Instance of 'Transaction'
    at Object.throw_ [as throw] (http://localhost:42399/dart_sdk.js:4332:11)
    at cloud_firestore_web.FirebaseFirestoreWeb.new.runTransaction (http://localhost:42399/packages/cloud_firestore_web/src/write_batch_web.dart.lib.js:719:21)
    at runTransaction.throw (<anonymous>)
    at http://localhost:42399/dart_sdk.js:37603:38
    at _RootZone.runBinary (http://localhost:42399/dart_sdk.js:37456:58)
    at _FutureListener.thenAwait.handleError (http://localhost:42399/dart_sdk.js:32440:48)
    at handleError (http://localhost:42399/dart_sdk.js:32991:51)
    at Function._propagateToListeners (http://localhost:42399/dart_sdk.js:33017:17)
    at _Future.new.[_completeError] (http://localhost:42399/dart_sdk.js:32864:23)
    at http://localhost:42399/dart_sdk.js:33087:38
    at _RootZone.runBinary (http://localhost:42399/dart_sdk.js:37456:58)
    at _FutureListener.then.handleError (http://localhost:42399/dart_sdk.js:32440:48)
    at handleError (http://localhost:42399/dart_sdk.js:32991:51)
    at Function._propagateToListeners (http://localhost:42399/dart_sdk.js:33017:17)
    at _Future.new.[_completeError] (http://localhost:42399/dart_sdk.js:32864:23)
    at async._AsyncCallbackEntry.new.callback (http://localhost:42399/dart_sdk.js:32902:31)
    at Object._microtaskLoop (http://localhost:42399/dart_sdk.js:37712:13)
    at _startMicrotaskLoop (http://localhost:42399/dart_sdk.js:37718:13)
    at http://localhost:42399/dart_sdk.js:33230:9
```

Reply:
I see it happens only on the web, I'll look into it

Reply:
The issue is the `return transaction.set`, that should not return (Line ~78)

Reply:
(It makes sense because runTransaction by default assumes it's a <void> transaction, I guess that returning something non-void is confusing. I'm going to create an issue, regardless.

Reply:
https://github.com/FirebaseExtended/flutterfire/issues/3495